### DISCUSSION TITLE
Releases

### DISCUSSION THREAD

**Discussion by scgm11** [asterisk-java/asterisk-java#486] [Announcements]
After updating deps and this 2 small fixes maybe we can do a release? @piotrooo

**Comment by piotrooo** [comment]
Yea, we can deploy 3.38.0

**Comment by piotrooo** [comment]
Done.
[3.38.0](https://github.com/asterisk-java/asterisk-java/releases/tag/3.38.0) released.

**Reply by scgm11** [reply]
is the maven central publish done automatically?

**Reply by piotrooo** [reply]
Yes. But it take some time to show it in central.

But you can download it.

https://repo1.maven.org/maven2/org/asteriskjava/asterisk-java/3.38.0/

**Reply by scgm11** [reply]
no need just wanted to know thanks!

**Reply by piotrooo** [reply]
Whole process is almost automatic. It's done by GitHub Actions and releases (with changelog).
